app/(gcforms)/[locale]/(form administration)/form-builder/components/shared/right-panel/RightPanel.tsx

```diff
@@ -28,9 +29,15 @@ const TabButton = ({
   onClick: () => void;
   className?: string;
 }) => {
+  const { t } = useTranslation("form-builder");
+  const [speak] = useLiveMessage();
+  const activate = () => {
+    onClick();
+    speak(t("rightPanel.activate", { panel: text }));
+  };
   const handleKeyDown = (e: React.KeyboardEvent<HTMLButtonElement>) => {
```

Comment:
Thinking we should call this loadPage or something that's more relevant used by more than keydown

Reply:
Maybe loadTab ?

Reply:
What's 

```
 { panel: text }
```

?

Reply:
"text" is the param passed into the component that labels the tab e.g. "Questions". Panel is what it's labeling, though I guess "tab" (vs panel) would be more right. Or combine them: "tabPanelLabel" :) into
`{tabPanelLabel: text}`

Reply:
ah right